mathics/builtin/system.py

```diff
@@ -871,3 +871,28 @@ def eval_with_symbol(self, symbol, evaluation: Evaluation) -> Integer:
         else:
             gc.collect()
             return Integer0
+
+
+class Breakpoint(Builtin):
+    """
+    <dl>
+      <dt>'Breakpoint[]'
+      <dd> Invoke a Python breakpoint. By default, the Python debugger \
+           (pdb) is loaded. For loading other debuggers, the Python environment \
+           variable `PYTHONBREAKPOINT` can be utilized.
+    </dl>
+
+    >> Breakpoint[]
```

Comment:
Add a pytest inside, which captures the interruption.

Reply:
To get doctests to work here is a pretty simple solution....

Add:

```
def disabled_breakpoint():
    return
```

somewhere like in `mathics.main` or `mathics.docpipeline` and then run using `PYTHONBREAKPOINT="mathics.main.disabled_breakpoint" ...`

Reply:
This might confuse the users of the function?

Reply:
> This might confuse the users of the function?

I am not sure what you mean. If you mean that someone sets PYTHONBREAKPOINT to this function that is intended to be used in testing, then things can be made more explicit by having `disabled_breakpoint()` print a message saying that a breakpoint was called but it is ignoring it. For example: 

```
def disabled_breakpoint():
   print("PYTHONBREAKPOINT is set to disabled_breakpoint. Ignoring breakpoint.")
```

Reply:
Using `PYTHONBREAKPOINT` is a great idea!